src/services/passwordGeneration.service.ts

```diff
@@ -314,9 +303,11 @@ export class PasswordGenerationService implements PasswordGenerationServiceAbstr
                 enforcedOptions = new PasswordGeneratorPolicyOptions();
             }
 
-            // Could be null if the policy options weren't adjusted with new data
-            if (currentPolicy.data.allowPassword != null && !currentPolicy.data.allowPassword) {
-                enforcedOptions.allowPassword = false;
+            // Password wins in multi-org collisions
+            if (currentPolicy.data.defaultType &&
+                currentPolicy.data.defaultType !== 'user' &&
```

Comment:
user?

Reply:
During policy edit I've added a dropdown selector for a default type. To maintain backwards compatibility and current use cases (user defines/selects the password TYPE within the generator) I've add a `User Preference` option so the org will not force a default type.

<img width="295" alt="Screen Shot 2020-03-10 at 9 32 35 AM" src="https://user-images.githubusercontent.com/26154748/76323404-ab8bfb80-62b2-11ea-9f41-a6a99af455da.png">

Reply:
Can we just handle that as `null`, meaning no default type is selected for this policy?

Reply:
That can be done - just went the explicit way for easy understanding of the code.  `null` or `' '` should fail the initial condition (was looking for a truthy response - did I do that correctly?)

Reply:
I would prefer to stay away from truthy evaluations if we can. Do explicit null check like `currentPolicy.data.defaultType != null`

Reply:
Alrighty - will update.